hs_swat_modelinstance/models.py

```diff
@@ -46,12 +46,18 @@ def get_swat_model_objectives(self):
 
     @classmethod
     def _add_swat_objective(cls, model_objective, objectives):
```

Comment:
@pkdash The validation was created for the users who are using the rest api to create the resource to make sure that they pass the correct string. The validation method validates the values from creating a new resourcre using the normal way or the rest api as both of them pass string values. After fixing the versioning problem, now these integer values pass but ignored in the validation method. Do we really need to validate  that? If yes, any suggestions to enhance this?